### PR TITLE
add mammoth@1.2.5 w/ npm auto-update

### LINKED ISSUE #9168: [Request] Add mammoth
**Library name:** mammoth
**Git repository url:** https://github.com/mwilliamson/mammoth.js
**npm package url(optional):** https://www.npmjs.com/package/mammoth
**License(s):** BSD-2-Clause
**Official homepage:** https://github.com/mwilliamson/mammoth.js
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t , please help me review this PR, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mwilliamson/mammoth.js
 * Official website (optional, not the repository): https://mike.zwobble.org/projects/mammoth/ 
* NPM package url (optional): https://www.npmjs.com/package/mammoth/ 
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 219
   - Count of watchers: 14
   - Count of forks: 45
 * NPM download stats (optional):
   - Downloads in the last day: 74
   - Downloads in the last week: 883
   - Downloads in the last month: 2868

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [X] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #9168, cc @mwilliamson

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -160,6 +160,13 @@ ajax/libs/Hyphenator/ersion_*
 ajax/libs/gemma/0.1.0/
 ajax/libs/gemma/0.4.0/
 ajax/libs/origamijs/0.0.2/
+ajax/libs/mammoth/0.1.0/
+ajax/libs/mammoth/0.1.1/
+ajax/libs/mammoth/0.1.2/
+ajax/libs/mammoth/0.1.3/
+ajax/libs/mammoth/0.1.4/
+ajax/libs/mammoth/0.1.5/
+ajax/libs/mammoth/0.1.6/
```
Comment (PeterDaveHello) [inline .gitignore]: Looks like we don't need theses mammoth since their files won't be grabbed by the current rule of auto-update.

**Comment by mwilliamson** [comment]
As I said, I would also recommend ignoring anything before 1.0.3.

**Comment by PeterDaveHello** [comment]
If the auto-update rule can not fetch the old versions, the rule for the versions in `.gitignore` is meaningless.

**Comment by nolsherry** [comment]
@PeterDaveHello, do you mean that we don't add those versions in `.gitignore` file?

**Comment by PeterDaveHello** [comment]
@nolsherry yes.

**Comment by nolsherry** [comment]
@pvnr0082t , I have removed `ajax/libs/mammoth/0.1.0`~`ajax/libs/mammoth/0.1.6` in `.gitignore` file, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Wrong PR title

**Pull request merged**